content/developer/reference/frontend/odoo_editor.rst

```diff
@@ -124,8 +124,7 @@ There are three ways to filter commands:
    :ref:`command <reference/frontend/odoo_editor/powerbox/command>`'s
    `isDisabled` entry.
 #. The user can filter commands by simply typing text after opening the
-   Powerbox. It will fuzzy-match that text with the names of the categories and
-   commands.
+   Powerbox. It will fuzzy-match that text with the names of the categories and commands, or match that text with the description of the commands.
```

Comment:
"description" -> "descriptions"
Also, reading this I understand that the matching with descriptions is not fuzzy, is that correct? If so, maybe specify to remove ambiguity (something like "or exact-match"). If not, maybe say "of the categories, commands, and their descriptions".

Reply:
Then I'd say go ahead and call it fuzzy ^^'

Reply:
@Zinston I went ahead and did that.
I've slightly changed the structure of the sentence as well, it seems clearer like this to me. What do you think ?

Reply:
Looks perfect to me, nice job :+1:

@odoo/javascript-doc-review Can you please validate this small PR?

Reply:
@dmo-odoo then do you prefer I call it "exact" match even though it's not an exact match either ?